python/lsst/obs/base/exposureAssembler.py

```diff
@@ -237,12 +229,19 @@ def assemble(self, components):
         info.setDetector(components.pop("detector", None))
         info.setTransmissionCurve(components.pop("transmissionCurve", None))
 
-        # Filter needs to be updated specially to match Exposure behavior
+        # A FilterLabel component gets read correctly even for version 1
+        # Exposure files, which persist the info as a Filter.
+        # TODO: deprecate in DM-27177, remove in DM-27811
```

Comment:
Oh, is this "TODO" is necessary here? This isn't the definition, just a call.

Reply:
I think so, since "deprecation" in this case would probably involve rewriting the code so that "filter" is the primary name again.

Reply:
I've reworded the comment to be less generic.